deploy/operator.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: antrea-operator
```

Comment:
Do you think we should have a separate namespace for operator, or we just use kube-system?

Reply:
Let's create a separate antrea-operator namespace for now

Reply:
Done. Set  namespace as `antrea-operator`.

Reply:
You also need a namespace yaml file

Reply:
Done.
- Add namespace yml
- Update role type as `ClusterRole`